com.github.rssguard.json

```diff
@@ -0,0 +1,34 @@
+{
+    "id": "com.github.rssguard",
+    "branch": "master",
```

Comment:
Flathub [requires the branch name to be 'stable' if present](https://github.com/flathub/flathub/wiki/Review-Guidelines) (you can just delete it).

Reply:
Author of RSS Guard here: So creating "stable" branch would be enough?

Reply:
The line specifying a branch can simply be removed.

Reply:
@martinrotter, `branch` field in the flatpak manifest is about ostree branch, not the git branch it's going to be build from. This way you can have multiple versions of app installed. (Not supported on Flathub though – we only take stable releases, thus such assumption.)